.github/workflows/go.yml

```diff
@@ -13,24 +13,23 @@ jobs:
 
   build:
     runs-on: ubuntu-latest
+    environment: Test
```

Comment:
@tazarov  If you wanto to use secrets in this Pull request, you can 
1. create a new environment in your github acitons  (environment `Test` )
<img width="1198" alt="image" src="https://github.com/amikos-tech/chroma-go/assets/30621793/c931fca5-db8b-4291-8d4b-fd220677f996">
2. create secrets inner that environment ( environment `Test` for example)
3.  set reviewers for this environment 
<img width="1091" alt="image" src="https://github.com/amikos-tech/chroma-go/assets/30621793/c8fcd5c4-98cf-466f-987c-d9b810156763">

4. use this environment in your workflow just like this `build` job

Reply:
When you trigger the workflow, this job need another manual approval so it can read secrets in that environment.